### PR TITLE
Updated data mapping of auction and winning bid objects in pubxai analytics adapter. 

### PR DESCRIPTION
**Pull request by pnh-pubx**
## Type of change
- [*] Bugfix
- [*] Refactoring (no functional changes, no api changes)

## Description of change
Updated winning bid and auction object fields that are being sent to the server.

### CODE REVIEW

**Comment by pnh-pubx** [comment]
Looks like the circleci build failure is related to something else.

File: modules/pubxaiAnalyticsAdapter.js [hunk 1]
```diff
@@ -134,8 +140,9 @@ pubxaiAnalyticsAdapter.shouldFireEventRequest = function (samplingRate = 1) {
 function send(data, status) {
   if (pubxaiAnalyticsAdapter.shouldFireEventRequest(initOptions.samplingRate)) {
     let location = utils.getWindowLocation();
-    if (typeof data !== 'undefined' && typeof data.auctionInit !== 'undefined') {
```
Comment (blingster7) [inline modules/pubxaiAnalyticsAdapter.js:137]: any reason why you took the `data.auctionInit !== undefined` check away?

**Review by blingster7 — APPROVED** [review]
LGTM

File: modules/pubxaiAnalyticsAdapter.js (hunk 1 above)
Comment (pnh-pubx) [reply modules/pubxaiAnalyticsAdapter.js:137]: I have removed auctionInit from the data object completely as we don't need it anymore. That's why this condition isn't required here.

**Pull request merged**